package.json

```diff
@@ -12,12 +12,22 @@
   "devDependencies": {
     "@types/chai": "^4.0.1",
     "@types/mocha": "^2.2.41",
+    "@types/webdriverio": "^4.7.0",
```

Comment:
I assume these dev-dependencies are all already present in the main repo?

Reply:
Yes, they are all taken from the main repository.

Reply:
ok, so presumably they have already been approved to be used by CQs.